package.json

```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "@algoan/nestjs-logging-interceptor": "^2.1.2",
+    "@celo/base": "^0.0.3",
```

Comment:
So we shouldn't need @celo/base anymore here.
In the larger PR I merged I changed the way we import dependencies from `celo/monorepo` because if import them via NPM threre's no way to link towards a specific git version and we would have to wait for a feature to be released before being able to use it. And it would block development I feel.

That's why I setup `celo/monorepo` as a submodule and and built that dependency locally.

For the first time setup fo the new flow you should follow the steps in the README. Specifically this part:
```
# `celo-monorepo` is a git submodule -- update downloads it in place
> git submodule update --init
# install `celo-monorepo` dependencies
> yarn deps:celo:install
> yarn deps:celo:build
```

After that the `@celo/base` dependency should be found throw the rewrite rule in `tsconfig.json`:
```
"@celo/*": [
        "libs/celo/packages/*"
      ]
```